lib/pages/game.dart

```diff
@@ -39,6 +39,7 @@ class GamePage extends StatelessWidget {
                               "payload": {
                                 "x": delta.dx,
                                 "y": delta.dy,
+                                "username": user.username,
```

Comment:
There is another event `BUTTON` on line 57, add there too.

Reply:
@sanket143 done.